pkg/scheduler/framework/plugins/interpodaffinity/filtering_test.go

```diff
@@ -54,7 +55,6 @@ func createPodWithAffinityTerms(namespace, nodeName string, labels map[string]st
 			},
```

Comment:
You can also use pod wrapper in `createPodWithAffinityTerms`.

Reply:
Thanks, converted `createPodWithAffinityTerms` to use `PodWrapper`